bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,304 @@
+should never be used in production.
+- `file`: Stores encrypted keys on disk, which is a more secure option than test but 
+less secure than using the OS keyring.
+- `os`: Uses the operating system's native keyring, providing the highest level of 
+security by relying on OS-managed encryption and access controls.
+
+The `<name>` specifies a unique identifier for the key.
+The `--home` flag specifies the directory where your node files will be stored 
+(e.g. `--home ./nodeDir`)
+
+The execution result displays the address of the newly generated key and its 
+public key. Following is a sample output for the command:
+
+```shell
+- address: bbn1kvajzzn6gtfn2x6ujfvd6q54etzxnqg7pkylk9
+  name: <name>
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey",
+           key: "Ayau+8f945c1iQp9tfTVaCT5lzhD8n4MRkZNqpoL6Tpo"}'
+  type: local
+```
+
+Make sure to securely store this information, particularly your address and 
+private key details. Losing access to these credentials would mean losing 
+control of your validator and any staked funds.
+
+## Keys for a BLS Voting
+### What is BLS Voting
+
+Babylon validators are required to participate in
+[BLS](https://en.wikipedia.org/wiki/BLS_digital_signature) voting
+at the end of each epoch.
+The Babylon blockchain defines epochs as a fixed period
+within the blockchain, defined or set by a number of blocks,
+during which the validator set remains consistent. 
+At the end of the epoch,
+the validator BLS signatures are aggregated to create a compact checkpoint
+that is timestamped on the Bitcoin ledger.
+The BLS voting mechanism achieves a significant reduction in the cost of
+checkpoints, while the epoching mechanism specifies a defined frequency
+for checkpointing in the Bitcoin blockchain.
+
+### Create BLS Key
+
+To generate your BLS key, you'll need to use your validator address from the previous step. 
+Run this command:
+
+```shell
+babylond create-bls-key <address>
+```
+
+Replace `<address>` with your validator address from the earlier keyring 
+generation (it should look similar to `bbn1kvajzzn6gtfn2x6ujfvd6q54etzxnqg7pkylk9`). 
+
+The system will automatically:
+1. Generate a new BLS key pair
+2. Associate it with your validator address
+3. Store it in your node's configuration file at 
+`~/.<path>/config/priv_validator_key.json`
+
+This key will be used automatically by your validator software when it needs 
+to participate in epoch-end signature collection. The BLS signatures help 
+create compact, efficient proofs of consensus that can be later timestamped to Bitcoin.
+
+Important: The `priv_validator_key.json` file contains sensitive key material. 
+Make sure to backup this file and store it securely, as it's essential for your 
+validator's operation and cannot be recovered if lost.
+
+## Prepare Validator Configuration
+
+Create your validator configuration file:
+
+```shell
+cat > <path>/config/validator.json << EOF
+{
+  "pubkey": $(cat pubkey.json),
```

Comment:
Where can I find the `pubkey.json`?
Looks like it's generated from `babylond tendermint show-validator` command